### PR TITLE
Add insight section

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Fixes # (issue)
Add insight section for stories page

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshots

screencapture-localhost-6006-iframe-html-2021-08-12-13_31_47

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/Card/index.js [hunk 1]
```diff
@@ -15,34 +10,34 @@ const Card = ({ href, children, image, ...props }) => {
   const classes = useStyles(props);
 
   return (
-    <MuiCard className={classes.root}>
```
Comment (kilemensi) [inline src/components/Card/index.js:18]: What's happening in this component?

**Review by kilemensi — COMMENTED** [review]
I'm kind of lost @nerdlet ? Why are renaming Stories to Insights ?

**Comment by nerdlet** [comment]
> I'm kind of lost @nerdlet. Why are renaming Stories to Insights ?

Um, we are not renaming stories ...this is for the insight section

File: src/components/Card/index.js (hunk 1 above)
Comment (nerdlet) [reply src/components/Card/index.js:18]: @kilemensi For this section, noticed that using cards drowns out box shadow applied on it for the insight class cards.So I used `div` instead to fix that

Comment (kilemensi) [reply src/components/Card/index.js:18]: `CardActionArea` is there to support clicking/linking to actual story so we should not remove it. You can disable button ripple effect [global](https://material-ui.com/getting-started/faq/#how-can-i-disable-the-ripple-effect-globally) and you should also be able to style `&:hover` on the component if need be.

**Comment by kilemensi** [comment]
> > I'm kind of lost @nerdlet. Why are renaming Stories to Insights ?
> 
> Um, we are not renaming stories ...this is for the insight section

I see 👇🏾 on GitHub i.e. `Stories` has been moved to `Insights`
Screenshot from 2021-08-12 15-48-07

**Comment by nerdlet** [comment]
> > > I'm kind of lost @nerdlet. Why are renaming Stories to Insights ?
> > 
> > 
> > Um, we are not renaming stories ...this is for the insight section
> 
> I see 👇🏾 on GitHub i.e. `Stories` has been moved to `Insights`
> Screenshot from 2021-08-12 15-48-07

😞 Let me look into it and fix that...

**Comment by kilemensi** [comment]
_Heita_ @nerdlet 

Is this component being used in any page?

**Pull request closed without merging**